PowerPointLabs/PowerPointLabs/ActionFramework/Content/EffectsLabBlurrinessContentHandler.cs

```diff
@@ -0,0 +1,52 @@
+﻿using PowerPointLabs.ActionFramework.Common.Attribute;
+using PowerPointLabs.ActionFramework.Common.Interface;
+
+namespace PowerPointLabs.ActionFramework.Content
+{
+    [ExportContentRibbonId(
+        TextCollection.EffectsLabBlurrinessFeatureSelected + TextCollection.DynamicMenuId,
+        TextCollection.EffectsLabBlurrinessFeatureRemainder + TextCollection.DynamicMenuId,
+        TextCollection.EffectsLabBlurrinessFeatureBackground + TextCollection.DynamicMenuId)]
+    class EffectsLabBlurrinessContentHandler : ContentHandler
+    {
+        protected override string GetContent(string ribbonId, string ribbonTag)
+        {
+            var feature = ribbonId.Replace(TextCollection.DynamicMenuId, "");
```

Comment:
Do you mean adding comments with the url to resources about dynamic menu? Do I add the comment in this file?